src/Footer/Footer.scss

```diff
@@ -0,0 +1,19 @@
+
+@media only screen and (max-width: 600px) {
+
+.footerMobile{
+    position: fixed;
+    width: 100%;
+    bottom: 0;
+    transform: translateY(30%);
```

Comment:
Does giving it a fixed height on all phones not play well with different widths? I've not got anything against this I'm just curious :)

Reply:
With width set to 100% and not specifying a height the height auto adjusts, because it is an image when we tested it with a set height the image was distorting

Reply:
Okay sure, sounds good, I imagine the default image behaviour just doesn't maintain aspect ratio very nicely!